roles/edpm_kernel/tasks/kernelargs.yml

```diff
@@ -100,13 +100,23 @@
         - /boot/efi/EFI/fedora
 
     - name: Generate grub config
-      ansible.builtin.command: "grub2-mkconfig -o /boot/grub2/grub.cfg"
+      ansible.builtin.command: >-
+        grub2-mkconfig -o /boot/grub2/grub.cfg
+        {{ '--update-bls-cmdline'
+        if ansible_facts['distribution'] == 'RedHat'
+        and ansible_facts['distribution_version'] | float >= 9.3
```

Comment:
so this should be 

ansible_facts['distribution_version'] is version( '9.3', '>=' )

https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_tests.html#comparing-versions

apparently using filter as a test is deprecated and raises a warning from ansibel 2.5

https://cn-ansibledoc.readthedocs.io/zh-cn/latest/user_guide/playbooks_tests.html#test-syntax